build.gradle.kts

```
@@ -55,73 +47,30 @@ kotlin {
         browser()
         nodejs()
     }
-    wasm {
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs {
         browser()
+        nodejs()
         binaries.executable()
     }
```

I think we can simplify as in the example below. What do you think? AFAIK These configurations are only for the final output.

```suggestion
    wasmJs()
```

---

If we can remove, also remove on JS please.

---

What do you mean when you say `These configurations are only for the final output.` ?

---

Ktor is a library and it defines the targets
https://github.com/ktorio/ktor/blob/6e52d76177a823b3c667989483e9bce35b04f53c/buildSrc/src/main/kotlin/TargetsConfig.kt#L45